src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,18 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+
+    private static final SessionFactory FACTORY = initFactory();
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        return FACTORY;
+    }
+
```

Comment:
Refer to the Common Mistakes:
<img width="1103" alt="image" src="https://github.com/mate-academy/hibernate-relations-hw/assets/140437083/cd34d976-8718-46cc-b73d-ddf7a225765d">